plugins/azure-devops-backend/src/service/router.ts

```diff
@@ -46,12 +46,13 @@ export async function createRouter(
   const token = config.getString('token');
   const host = config.getString('host');
   const organization = config.getString('organization');
+  const reader = UrlReaders.default(options);
```

Comment:
Do not create this yourself; add it to the router options above instead and pass in the one that's already in the backend environment https://github.com/backstage/backstage/blob/87649a06bfc38d39646117f4e5f80cd3ddf9d724/packages/backend/src/types.ts#L37

This will make it a breaking change for the backend package, so it would want to be bumped to minor in the changeset